video_service/internal/models/video.go

```diff
@@ -354,6 +354,18 @@ func (v *VideoModel) generateVideoListSQL(direction videoproto.SortDirection, pa
 		case videoproto.SortDirection_desc:
 			ds = ds.Order(goqu.I("rating").Desc())
 		}
+
+	case videoproto.OrderCategory_my_ratings:
+		ds = ds.Join(
```

Comment:
This is overall very clean, and it's how I would've done things; my only point of contention here is whether we should be using a left join here. I think it's okay as is, though